packages/core/test/helper/stream-helper.js

```diff
@@ -0,0 +1,23 @@
+import { eq } from '@briancavalier/assert'
+import { reduce } from '../../src/combinator/reduce'
+
+export function assertSame (s1, s2) {
+  return Promise.all([toArray(s1), toArray(s2)]).then(arrayEquals)
+}
+
+export function expectArray (array, s) {
+  return toArray(s).then(function (result) {
```

Comment:
oops, I think I missed this opportunity for partial application!  Can you update this to:

```js
return toArray(s).then(eq(result))
```

Thanks!